tests/Command/GenerateObjectAclCommandTest.php

```diff
@@ -112,6 +115,7 @@ public function testExecuteWithManipulatorNotObjectAclManipulatorInterface(): vo
             'sonata.admin.manipulator.acl.object.bar' => new \stdClass(),
         ];
 
+        // @phpstan-ignore-next-line
         $command = new GenerateObjectAclCommand($pool, $aclObjectManipulators);
```

Comment:
Will be nice to resolve it in other way but we can accept it.

Reply:
When we want to pass a wrong argument and see we're having an exception, I don't see another way